pkg/booleanpolicy/policyversion/version.go

```diff
@@ -37,12 +36,6 @@ func CurrentVersion() PolicyVersion {
 	return PolicyVersion{versions[len(versions)-1]}
```

Comment:
Regardless on whether we want to keep the policy check in the Central<->Sensor communication, most of this file, except maybe `CurrentVersion()`, can go away.